pcsx2/gui/MainFrame.cpp

```diff
@@ -584,19 +587,66 @@ void MainEmuFrame::OnActivate( wxActivateEvent& evt )
 }
 // ----------------------------------------------------------------------------
 
+static void GetMenuItemLabel(MenuIdentifiers Id, wxString* label)
```

Comment:
> It would probably be better to just use the switch directly instead of placing the switch in a function.

I'm not keen on separating it from a dedicated function, it looks much cleaner to handle the menu item labels naming separately and helps to avoid congesting up the code in ``MainEmuFrame::ApplyCoreStatus()``.